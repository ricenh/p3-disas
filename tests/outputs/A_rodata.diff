--- outputs/A_rodata.txt	2023-11-06 15:39:32.706109647 -0500
+++ expected/A_rodata.txt	2023-10-10 11:54:35.000000000 -0400
@@ -1 +1,4 @@
 Disassembly of data contents:
+  0x200:                               | .pos 0x200 rodata
+  0x200: 68 65 6c 6c 6f 00             |   .string "hello"
+
