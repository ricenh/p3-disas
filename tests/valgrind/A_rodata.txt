==1080569== Memcheck, a memory error detector
==1080569== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==1080569== Using Valgrind-3.18.1 and LibVEX; rerun with -h for copyright info
==1080569== Command: ../y86 -D inputs/rodata.o
==1080569== 
Disassembly of data contents:
==1080569== 
==1080569== HEAP SUMMARY:
==1080569==     in use at exit: 0 bytes in 0 blocks
==1080569==   total heap usage: 4 allocs, 4 frees, 20,952 bytes allocated
==1080569== 
==1080569== All heap blocks were freed -- no leaks are possible
==1080569== 
==1080569== For lists of detected and suppressed errors, rerun with: -s
==1080569== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
