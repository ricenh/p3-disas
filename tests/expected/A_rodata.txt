Disassembly of data contents:
  0x200:                               | .pos 0x200 rodata
  0x200: 68 65 6c 6c 6f 00             |   .string "hello"

